src/test/java/hw2/ex2/DifElementsPageTest.java

```diff
@@ -0,0 +1,92 @@
+package hw2.ex2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class DifElementsPageTest {
+
+    private WebDriver driver;
+
+    @BeforeMethod(alwaysRun = true)
+    public void ChromeSetUp()
+    {
+        driver = new ChromeDriver();
+    }
+
+    @Test (description = "Different Elements Page Test")
+    public void MyTest_02()  {
+
+        driver.manage().window().maximize();
+
+        //1. Open test site by URL
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+
+        //2. Assert Browser title
+        assertEquals(driver.getTitle(), "Home Page");
+
+        //3. Perform login
+        WebDriverWait waitcicon = new WebDriverWait(driver, 10);
+        waitcicon.until(ExpectedConditions.presenceOfElementLocated(By.id("user-icon")));
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys("Roman");
+        driver.findElement(By.id("password")).sendKeys("Jdi1234");
+        driver.findElement(By.id("login-button")).click();
```

Comment:
Maybe we can create a specific method for login?

Reply:
moved to separate method